### DISCUSSION TITLE
How to resize columns of core.Table ?

### DISCUSSION THREAD

**Discussion by heejit** [cogentcore/core#1245] [General]
How to resize column of core.Table in code and interactively (using mouse) ?

**Comment by kkoreilly** [comment]
We do not currently really support this feature, but it should be relatively easy to implement it. We can add support for doing it in code soon, and support for interactive resizing using the mouse after that. For now, you can use the `width` struct tag on field(s) in your table struct type to change the width of columns. For example, you could do something like `width:"80"`.

**Reply by heejit** [reply]
okay fine.
